app/src/app/[lng]/[inventory]/InventoryResultTab/EmissionsForecast/GrowthRatesExplanationModalTable.tsx

```diff
@@ -18,6 +18,12 @@ export const GrowthRatesExplanationModalTable = ({
   growthRates: ProjectionData;
   t: TFunction;
 }) => {
+  function getNameTranslationString(
+    sector: ISector | { name: string; color: string; referenceNumber: string },
+  ) {
+    return sector.name === "ippu" ? sector.name + "-short" : sector.name;
```

Comment:
Should we change the default string for this sector to be the short one? So we don't have to make this special case just for this one sector...

Reply:
we use the short name in only one place, while we use the long name everywhere else, I think making the change would require a lot of code changes and possible bugs